src/combine_hru_quantiles.slurm

```diff
@@ -1,15 +1,17 @@
 #!/bin/bash
 #SBATCH --job-name=combine_quantiles     # name that you chose
-#SBATCH -c 1                   # number of cores per task
-#SBATCH -p UV              # the partition you want to use, for this case prod is best
-#SBATCH -A iidd                 # your account
+#SBATCH -c 1                            # number of cores per task
+#SBATCH -p UV,normal              # choose both partitions
```

Comment:
interesting, does this just mean you get allocated whichever is available?

Reply:
yes, that's the impression that Natalya gave, I haven't gone digging through their docs again to look for more details on it

Reply:
Cool, hadn't seen that before